src/image/components/runtime.tsx

```diff
@@ -33,17 +48,88 @@ export const Runtime: React.FC<IProps> = ({ authoredState, report }) => {
   const getOriginalImageSize = (e: any) => {
     if (e.target && e.target.naturalWidth && e.target.naturalHeight) {
       imageSize.current = { width: e.target.naturalWidth, height: e.target.naturalHeight };
-      const aspectRatio = e.target.naturalWidth / e.target.naturalHeight;
-      setSupportedFeatures({
-        interactiveState: true,
-        authoredState: true,
-        aspectRatio
-      });
+
+      if (authoredState.scaling === "fitWidth") {
+        if (!baseContainer.current) {
+          // FIMXE:
+          // If the image loads before the baseContainer is initialized this will
+          // default to LARA default aspect ratio of 1.3
+          // That will result in a clipped image in most cases.
+          // A better solution would be to delay the execution of this method
+          // until both the image is loaded and the baseContainer is available
+          console.error("image interactive baseContainer not available. aspectRatio will be wrong");
```

Comment:
Have you ever had this problem / seen this error message?
I guess this check won't hurt and might help to debug some issues in the future. But I'll be surprised to see it.

Reply:
Yeah I didn't see, but it seemed theoretically possible, so I added the error just incase.